src/Domain/CoordinateMaps/TimeDependent/Translation.hpp

```diff
@@ -11,28 +11,93 @@
 #include <unordered_map>
 
 #include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/MathFunctions/MathFunction.hpp"
 #include "Utilities/TypeTraits/RemoveReferenceWrapper.hpp"
 
 /// \cond
-namespace domain {
-namespace FunctionsOfTime {
+namespace domain::FunctionsOfTime {
 class FunctionOfTime;
-}  // namespace FunctionsOfTime
-}  // namespace domain
+}  // namespace domain::FunctionsOfTime
 namespace PUP {
 class er;
 }  // namespace PUP
 /// \endcond
 
-namespace domain {
-namespace CoordinateMaps {
-namespace TimeDependent {
+namespace domain::CoordinateMaps::TimeDependent {
 /*!
  * \ingroup CoordMapsTimeDependentGroup
- * \brief Translation map defined by \f$\vec{x} = \vec{\xi}+\vec{T}(t)\f$.
+ * \brief Translation map defined by \f$\vec{x} = \vec{\xi}+F(r)\vec{T}(t)\f$.
  *
- * The map adds a translation, \f$\vec{T}(t)\f$, to the coordinates
- * \f$\vec{\xi}\f$, where \f$\vec{T}(t)\f$ is a FunctionOfTime.
+ * \details The map adds a translation, \f$F(r)\vec{T}(t)\f$, to the coordinates
+ * \f$\vec{\xi}\f$, where \f$\vec{T}(t)\f$ is a FunctionOfTime and\f$F(r)\f$ is
+ * a 1D radial MathFunction. The radius of each point is found by subtracting
+ * the center map argument from the coordinates \f$\vec{\xi}\f$ or the target
+ * coordinates \f$\vec{\bar{\xi}}\f$. The Translation Map class is overloaded so
+ * that the user can choose if a radial depedence is needed for their problem.
+ * If a radial dependence is not specified, this sets \f$F(r) = 1\f$.
+ *
+ * ### Mapped Coordinates
+ *
+ * The translation map translates the coordinates \f$\vec{\xi}\f$ to the
+ * target coordinates
+ * \f{equation}{
+ * \vec{\bar{\xi}} = \vec{\xi} + F(r)\vec{T}(t)
+ * \f}
+ *
+ * If you only supply a FunctionOfTime to the constructor of this class, the
+ * radial function will be set to 1.0 causing a uniform translation for your
+ * coordinates. If a FunctionOfTime, MathFunction, and map center are passed in,
+ * the radius will be found through
+ * \f{equation}{
+ * r = |\vec{\xi} - \vec{c}|
+ * \f}
+ * where r is the radius and \f$\vec{c}\f$ is the center argument.
+ *
+ * ### Inverse Translation
+ * The inverse translation translates the coordinate \f$\vec{\bar{\xi}}\f$ to
+ * the original coordinates
+ * \f{equation}{
+ * \vec{\xi} = \vec{\bar{\xi}} - F(root)\vec{T}(t)
```

Comment:
`root` renders in $\LaTeX$ making it look weird. Either use a variable for it and say it's the root of `F`, or wrap it in `\mathrm{root}`.